src/monattack.h

```diff
@@ -89,6 +89,7 @@ void rifle              (monster *z, Creature *target); // Automated M4
 void frag               (monster *z, Creature *target); // Automated MGL
 void tankgun            (monster *z, Creature *target); // Tankbot primary.
 void flame              (monster *z, Creature *target);
+void lay_egg            ( monster *z );
```

Comment:
Move it up to the attacks above. It needs to be `bool lay_egg( monster *z);` or it won't be bindable properly.